build.sbt

```diff
@@ -10,7 +10,7 @@ lazy val appName: String = "pension-scheme-return-frontend"
 addCompilerPlugin("org.typelevel" % "kind-projector" % "0.13.2" cross CrossVersion.full)
 
 lazy val root = (project in file("."))
-  .enablePlugins(PlayScala, SbtAutoBuildPlugin, SbtDistributablesPlugin)
+  .enablePlugins(PlayScala, SbtDistributablesPlugin)
```

Comment:
Why do we need this?

Reply:
https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=TEC&postingDay=2021%2F5%2F5&title=New+version+of+sbt-auto-build
we don't need to enable this, it's auto enabled